### ISSUE TITLE
Missing js files in socket-apollo-link?

### ISSUE THREAD

**Issue by rafbgarcia** [absinthe-graphql/absinthe-socket#3]
Hello there, trying to integrate and got this error

![ionic_app](https://user-images.githubusercontent.com/1904314/32556851-ad15eab8-c487-11e7-8162-586dde68bef9.png)

Tried to find the file but it seems to be missing?

```bash
➜  socket-apollo-link git:(master) ✗ pwd
/Users/rafa/project/node_modules/@absinthe/socket-apollo-link
➜  socket-apollo-link git:(master) ✗ tree
.
├── LICENSE.txt
├── README.md
├── flow-typed
│   └── npm
│       └── react-relay_v1.x.x.js
└── package.json

2 directories, 4 files
```

Thanks

**Comment by mgtitimoli** [comment]
Hi @rafbgarcia, thanks for having set some time to report this issue.

We are about to release a new version which includes this fix.

**Comment by mgtitimoli** [comment]
Fixed on Release 0.1.2
